src/components/Navbar/WalletMenu.tsx

```diff
@@ -42,6 +59,18 @@ export default function WalletMenu({ userAddress }: { userAddress: string }) {
     </>
   )
 
+  const Referral = () => (
```

Comment:
**suggestion** Could we move this component into it's own file?

Reply:
sure, let me update that.

Reply:
done